languages/elixir/exercises/concept/lists/.docs/instructions.md

```diff
@@ -0,0 +1,25 @@
+In this exercise you need to implement some functions to manipulate a list of programming languages.
```

Comment:
The tasks in this file are missing examples of the function calls. See [the booleans exercise](https://github.com/exercism/v3/blob/master/languages/elixir/exercises/concept/booleans/.docs/instructions.md#1-define-if-pac-man-eats-a-ghost) for an example.